lottie/src/main/java/com/airbnb/lottie/network/Fetcher.java

```diff
@@ -0,0 +1,15 @@
+package com.airbnb.lottie.network;
+
+import java.io.IOException;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.WorkerThread;
+
+/**
+ * Interface for custom fetcher
+ */
+public interface Fetcher {
```

Comment:
Let's name this `LottieNetworkFetcher` to make it explicit.

Reply:
done